query/control/controller_test.go

```diff
@@ -231,7 +241,8 @@ func TestController_QueryRuntimeError(t *testing.T) {
 			if stats.CompileDuration == 0 {
 				t.Error("expected compile duration to be above zero")
 			}
-			if stats.QueueDuration == 0 {
+			// No good way to slow down the queue duration for Windows-friendliness.
+			if runtime.GOOS != "windows" && stats.QueueDuration == 0 {
```

Comment:
I'd be fine with just checking compile duration and completely removing the queue duration check - as long as we check flux is populating at least some of the stats correctly